### PR TITLE
upped scikit-image and some naming changes

### PR DESCRIPTION
**Pull request by WillemSpek**
see #613 it's just some very minor changes that should enhance compatibility and not impede functionality. Should be a quick review :)

### CODE REVIEW

**Review by stefsmeets — APPROVED** [review]
What exactly does this fix?

**Comment by WillemSpek** [comment]
Thanks for the review! 
1. Dependency coonflict

`ERROR: Cannot install -r requirements.in (line 3) and -r requirements.in (line 7) because these package versions have conflicting dependencies.`

`The conflict is caused by:`
    `dianna 0.9.0 depends on scikit-image==0.19.1`
    `quantus 0.4.0 depends on scikit-image>=0.19.3`
    
2. problem with Quantus and DIANNA integration:

`TypeError: explain() got multiple values for argument 'model'`

Basically because quantus also uses model to pass on to the explain function which collides with the required positional argument model from DIANNA in kernelshap.explain

**Comment by stefsmeets** [comment]
Thanks for the explanation, makes sense :+1:
